README.md

```diff
@@ -1 +1,3 @@
-# 12-03-01-JUnit
\ No newline at end of file
+# 12-03-01-JUnit
+
+## Branch: LHN
```

Comment:
Bitte entfernen, da es sonst in den Main Branch überführt wird

Reply:
resolved: removed